src/shared/icons/Logo.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+// TODO: add props for controlling color and size
+const Logo = () => {
+  return (
+    <>
+      {/* <?xml version="1.0" encoding="iso-8859-1"?> */}
+      {/* <!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools --> */}
+      <svg
+        height="80px"
+        width="80px"
+        version="1.1"
+        id="Layer_1"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
+        viewBox="0 0 512 512"
+        xmlSpace="preserve">
```

Comment:
The logo now works. I had to change:
`xmlns:xlink` -> `xmlnsXlink`
`xml:space` -> `xmlSpace`

Because colons in attribute names are not valid react (JSX) syntax. This is just some wierd quirk you learn after working a while.

Reply:
@SahilRw